ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/HostStatusEntity.java

```diff
@@ -67,17 +63,18 @@ public boolean equals(final Object o) {
     }
 
     final HostStatusEntity that = (HostStatusEntity) o;
-    return Objects.equals(hostInfoEntity, that.hostInfoEntity)
-        && hostAlive == that.hostAlive && lastStatusUpdateMs == that.lastStatusUpdateMs;
+    return hostAlive == that.hostAlive
+        && lastStatusUpdateMs == that.lastStatusUpdateMs
+        && Objects.equals(hostStoreLags, that.hostStoreLags);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(hostInfoEntity, hostAlive, lastStatusUpdateMs);
+    return Objects.hash(hostAlive, lastStatusUpdateMs, hostStoreLags);
   }
 
   @Override
   public String toString() {
-    return hostInfoEntity + "," + hostAlive + "," + lastStatusUpdateMs;
+    return hostAlive + "," + lastStatusUpdateMs;
```

Comment:
nit: let's use the standard pattern here as well

Reply:
I don't understand what is wrong here

Reply:
The standard pattern is:
```java
  @Override
  public String toString() {
    return "FormatInfo{"
        + "format=" + format
        + ", fullSchemaName=" + fullSchemaName
        + ", delimiter=" + delimiter
        + '}';
  }
```
This (1) gives you the class name, and (2) gives you the field names for each. IntelliJ can automatically generate this for you.